docs/adrs/0008-appliance-calculator-location.md

```diff
@@ -0,0 +1,83 @@
+- [citizensadvice/smart-meter-tool](https://github.com/citizensadvice/smart-meter-tool) (has specific contractual deployment and hosting isolation requirements)
+- [citizensadvice/energy-comparison-table](https://github.com/citizensadvice/energy-comparison-table) (single rails app handling requests for the customer service rating comparison table)
+
+The appliance calculator itself will be rendered into an iframe on an advice collection page by the public website, which involves creating a [new content type](https://app.contentful.com/spaces/vms0u05139aw/environments/qa/content_types/interactiveTool/fields) to represent the calculator, making content-api understand that new content type and creating a new renderer in the public website codebase.  Rough implementation spikes have proved that this will work:
+- https://github.com/citizensadvice/content-api/tree/exp/interactive-tool-spike
+- https://github.com/citizensadvice/public-website/tree/exp/interactive-tool-spike
+
+## Decision
+
+1. Generalise from `energy-comparison-table` to `energy-apps`, by:
+   1. recreating the current app infrastructure with the more generic `energy-apps` name
+   2. switching to the the new sub-domain in production in the CloudFront config
+   3. renaming the `energy-comparison-table` app and namespacing the comparison table app appropriately
+2. Build the appliance calculator in the newly renamed `energy-apps` rails app
+
+## Consequences
+
+### Positive
+
+- all energy apps (excluding the smart meter tool) are contained in the same place
+- we will only need to solve certain problems once for all energy apps (eg monitoring, authentication, header and footer, country route handling)
+- little to no additional maintenance burden
+- can leverage the existing build and deployment workflows (which are robust and fast)
+- increased exposure to CDK and infrastructure configration for the energy team developer (the current infrastructure code was developed by the SRE team)
+
+
+### Negative
+
+- the additional work in re-creating and switching is significant, but this is ameliorated by the following factors:
+  - there is time to do this work now whilst the energy team wait for decisions about upcoming work (changes to the comparison table and the smart meter tool)
+  - it is easier to do now before more complexity is added to the single energy app
+- additional work in namespacing and organising the existing energy comparison table code
+
+## Discounted alternatives
+
+### Building the appliance calculator in public website
+
+The appliance calculator could also be built inside the public website codebase.
+
+#### Positives
+
+- problems like monitoring, authentication, header and footer, country route handling are all aready solved here
+- potentially easier / quicker for non-energy team devs to debug and support the appliance calculator code if required
+- little to no maintenance burdern
+
```

Comment:
Is there an argument that we're just interpreting a new content type from Contentful, and that this is conceptually what content-api & public website already do?

Reply:
Yep - and that code would still stay in public website and content api as part of this work, regardless of which option we choose here.  I'll write another ADR about that for clarity